test/ngAria/ariaSpec.js

```diff
@@ -616,10 +616,35 @@ describe('$aria', function() {
   describe('tabindex', function() {
     beforeEach(injectScopeAndCompiler);
 
-    it('should attach tabindex to role="checkbox", ng-click, and ng-dblclick', function() {
+    it('should not attach to native controls', function() {
+      var element = [
+        $compile("<button ng-click='something'></button>")(scope),
+        $compile("<a ng-href='#/something'>")(scope),
+        $compile("<input ng-model='val'>")(scope),
+        $compile("<textarea ng-model='val'></textarea>")(scope),
+        $compile("<select ng-model='val'></select>")(scope),
+        $compile("<details ng-model='val'></details>")(scope)
+      ];
+      expectAriaAttrOnEachElement(element, 'tabindex', undefined);
+    });
+
+    it('should not attach to rando ng-model elements', function() {
```

Comment:
1. `rando`  -->  `random` (?)

2. Why not ? If they have `ngModel` then they are supposed to have two-way binding (aka expect user input), so why not get a tabindex ?